plugin.audio.rautemusik/addon.xml

```diff
@@ -1,25 +1,33 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="plugin.audio.rautemusik"
-	name="RauteMusik"
-	version="0.3.0"
-	provider-name="Leo Moll">
+	name="RauteMusik.FM"
+	version="0.5.0"
+	provider-name="RauteMusik GmbH, Leo Moll">
 	<requires>
-		<import addon="xbmc.python" version="2.1.0"/>
+		<import addon="xbmc.python" version="2.14.0"/>
+		<import addon="xbmc.addon" version="13.0.0"/>
+		<import addon="xbmc.json" version="6.6.0"/>
 	</requires>
 	<extension point="xbmc.python.pluginsource" library="addon.py">
 		<provides>audio</provides>
 	</extension>
 	<extension point="xbmc.addon.metadata">
-		<summary lang="de">RauteMusik</summary>
-		<summary lang="en">RauteMusik</summary>
+		<summary lang="de">RauteMusik.FM</summary>
+		<summary lang="en">RauteMusik.FM</summary>
 		<description lang="de">Das Internetradio RauteMusik.FM wurde im Jahr 2003 gegründet und hat sich seitdem zu Europas größtem Online Radio entwickelt. Rund 300 Mitglieder sorgen täglich dafür, dass rund um die Uhr die beste Musik der vergangenen Jahre und die Hits aus den aktuellen Charts über das Webradio gesendet werden können.</description>
 		<description lang="en">RauteMusik.FM was founded in 2003 and has become one of the largest internet radio networks in Europe. More than 300 members are working hard in providing the best music of the last years and the current charts.</description>
+		<disclaimer lang="de">Enjoy our music!</disclaimer>
+		<disclaimer lang="en">Enjoy our music!</disclaimer>
 		<platform>all</platform>
-		<language>de</language>
+		<language>en de</language>
 		<license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
```

Comment:
This refers to content language. Does the addon provide english audio or is it only german?

Reply:
Difficult to say. The radio stations are located in Germany but the content is only music. There are a few ads in between and these are mainly spoken in German. In any case I see that it was a mistake to change this tag. Should I add a fix to this pull request?

Reply:
Fixed.